src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -827,11 +860,11 @@ private List<String> validateCli(ParseResult parseResult, List<File> filesToProc
             }
             // ensure there is no conflicting options
             else if (printAst || printAstWithComments || printJavadocTree || printTreeWithJavadoc
-                || xpath != null) {
+                || xpath != null || customTreePrinterClass != null) {
                 if (suppressionLineColumnNumber != null || configurationFile != null
                         || propertiesFile != null || outputPath != null
                         || parseResult.hasMatchedOption(OUTPUT_FORMAT_OPTION)) {
-                    result.add("Option '-t' cannot be used with other options.");
+                    result.add("Tree Options cannot be used with other options.");
```

Comment:
This wording was identified as out of date and confusing for options where `-t` is not used.

Examples:
````
PS M:\> java -jar .\checkstyle-10.5.0-all.jar -s out -t .
Option '-t' cannot be used with other options.
PS M:\> java -jar .\checkstyle-10.5.0-all.jar -s out -T .
Option '-t' cannot be used with other options.
PS M:\> java -jar .\checkstyle-10.5.0-all.jar -s out -J .
Option '-t' cannot be used with other options.
````